### PR TITLE
37 component add api's for new integration

### PR DESCRIPTION
**Pull request by ramtob**

### CODE REVIEW

File: src/app/@ansyn/map-facade/effects/map.effects.ts [hunk 1]
```diff
@@ -238,6 +238,24 @@ export class MapEffects {
 		})
 	);
 
+	@Effect({ dispatch: false })
+	setMapPositionByRect$ = this.actions$.pipe(
+		ofType<SetMapPositionByRectAction>(MapActionTypes.SET_MAP_POSITION_BY_RECT),
+		switchMap(({ payload: { id, rect } }: SetMapPositionByRectAction) => {
+			const communicator = this.communicatorsService.provide(id);
```
Comment (asafMasa) [inline src/app/@ansyn/map-facade/effects/map.effects.ts]: check if communicator exists

File: src/app/@ansyn/map-facade/effects/map.effects.ts [hunk 2]
```diff
@@ -238,6 +238,24 @@ export class MapEffects {
...
+			return communicator.setPositionByRect(rect);
+		})
+	);
+
+	@Effect({ dispatch: false })
+	setMapPositionByRadius$ = this.actions$.pipe(
+		ofType<SetMapPositionByRadiusAction>(MapActionTypes.SET_MAP_POSITION_BY_RADIUS),
+		switchMap(({ payload: { id, center, radiusInMeters } }: SetMapPositionByRadiusAction) => {
+			const communicator = this.communicatorsService.provide(id);
```
Comment (asafMasa) [inline src/app/@ansyn/map-facade/effects/map.effects.ts]: check if communicator exists

File: src/app/app/sendbox/sendbox/sendbox.component.html [hunk 3]
```diff
@@ -1,3 +1,7 @@
-<p>
-  sendbox works!
-</p>
+
```
Comment (asafMasa) [inline src/app/app/sendbox/sendbox/sendbox.component.html]: sand box not "send box" ?

File: src/app/@ansyn/map-facade/effects/map.effects.ts (hunk 1 above)
Comment (ramtob) [reply src/app/@ansyn/map-facade/effects/map.effects.ts]: done

File: src/app/@ansyn/map-facade/effects/map.effects.ts (hunk 2 above)
Comment (ramtob) [reply src/app/@ansyn/map-facade/effects/map.effects.ts]: done

File: src/app/app/sendbox/sendbox/sendbox.component.html (hunk 3 above)
Comment (ramtob) [reply src/app/app/sendbox/sendbox/sendbox.component.html]: Fixed

File: src/app/@ansyn/imagery/communicator-service/communicator.service.spec.ts [hunk 4]
```diff
@@ -46,7 +46,7 @@ describe('ImageryCommunicatorService', () => {
 		expect(imageryCommunicatorService.instanceCreated.emit).toHaveBeenCalledWith({ id: '1' });
 	});
 
-	fit('provide Communicator should return expected communicator', () => {
+	it('provide Communicator should return expected communicator', () => {
```
Comment (asafMasa) [inline src/app/@ansyn/imagery/communicator-service/communicator.service.spec.ts]: :+1:

File: src/app/@ansyn/map-facade/effects/map.effects.ts [hunk 5]
```diff
@@ -243,7 +243,8 @@ export class MapEffects {
 		ofType<SetMapPositionByRectAction>(MapActionTypes.SET_MAP_POSITION_BY_RECT),
 		switchMap(({ payload: { id, rect } }: SetMapPositionByRectAction) => {
 			const communicator = this.communicatorsService.provide(id);
-			return communicator.setPositionByRect(rect);
+			const result$ = communicator ? communicator.setPositionByRect(rect) : of(null);
```
Comment (asafMasa) [inline src/app/@ansyn/map-facade/effects/map.effects.ts]: EMPTY

File: src/app/@ansyn/map-facade/effects/map.effects.ts [hunk 6]
```diff
@@ -252,7 +253,8 @@ export class MapEffects {
 		ofType<SetMapPositionByRadiusAction>(MapActionTypes.SET_MAP_POSITION_BY_RADIUS),
 		switchMap(({ payload: { id, center, radiusInMeters } }: SetMapPositionByRadiusAction) => {
 			const communicator = this.communicatorsService.provide(id);
-			return communicator.setPositionByRadius(center, radiusInMeters);
+			const result$ = communicator ? communicator.setPositionByRadius(center, radiusInMeters) : of(null);
```
Comment (asafMasa) [inline src/app/@ansyn/map-facade/effects/map.effects.ts]: EMPTY

File: src/app/@ansyn/imagery/communicator-service/communicator.service.spec.ts (hunk 4 above)
Comment (ramtob) [reply src/app/@ansyn/imagery/communicator-service/communicator.service.spec.ts]: 👍

File: src/app/@ansyn/map-facade/effects/map.effects.ts (hunk 5 above)
Comment (ramtob) [reply src/app/@ansyn/map-facade/effects/map.effects.ts]: done

File: src/app/@ansyn/map-facade/effects/map.effects.ts (hunk 6 above)
Comment (ramtob) [reply src/app/@ansyn/map-facade/effects/map.effects.ts]: done

**Pull request merged**